package.json

```diff
@@ -3,16 +3,19 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./app.js"
+    "start": "nodemon ./app.js"
   },
   "dependencies": {
+    "bcrypt-nodejs": "0.0.3",
     "cookie-parser": "~1.4.3",
     "debug": "~2.6.9",
     "express": "~4.16.0",
     "hbs": "~4.0.1",
     "helmet": "^3.13.0",
     "http-errors": "~1.6.2",
     "mongoose": "^5.3.1",
-    "morgan": "~1.9.0"
+    "mongoose-validator": "^2.1.0",
+    "morgan": "~1.9.0",
+    "nodemon": "^1.18.4"
```

Comment:
Nodemon shouldn't be in dependencies.   The same reason.  You can install nodemon globally   "npm install -g nodemon" but shouldn't be in dependencies in the project.